dbaccess/transaction_output.go

```diff
@@ -47,14 +47,13 @@ func TransactionOutputsByOutpoints(ctx Context, outpoints []*Outpoint) ([]*dbmod
 
 	var dbPreviousTransactionsOutputs []*dbmodels.TransactionOutput
 	// fetch previous transaction outputs in chunks to prevent too-large SQL queries
-	for offset := 0; offset < len(outpointTuples); {
+	for i, offset := 0, 0; offset < len(outpointTuples); i++ {
```

Comment:
I'd prefer if you didn't make for loops this fancy. Please put i and i++ on different lines.